src/blocks/donate/edit.js

```diff
@@ -255,10 +295,15 @@ class Edit extends Component {
 			<Fragment>
 				{ form }
 				<InspectorControls>
-					<PanelBody title={ __( 'Donate Block' ) }>
-						<p>{ __( 'The Donate Block allows you to collect donations from readers. The fields are automatically defined based on your donation settings.' ) }</p>
-						<ExternalLink href='/wp-admin/admin.php?page=newspack-donations-wizard#/'>
-							{ __( 'Edit donation settings.' ) }
+					<PanelBody title={ __( 'Donate Block', 'newspack-blocks' ) }>
+						<p>
+							{ __(
+								'The Donate Block allows you to collect donations from readers. The fields are automatically defined based on your donation settings.',
+								'newspack'
```

Comment:
Should this be `newspack-blocks`?

```suggestion
								'newspack-blocks'
```

Reply:
D'oh, thanks Jeff! Fixed in ae290c3.